src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -1292,7 +1294,7 @@ private void UpdateWindowProperties(WindowProperties newProperties, bool forceCh
                 if (IsWindowVisible(_hwnd))
                     style |= WindowStyles.WS_VISIBLE;
 
-                if (newProperties.IsResizable)
+                if (newProperties.IsResizable || newProperties.WindowState == WindowState.Maximized)
```

Comment:
Does it handle fullscreen properly? I think we are expected to handle it as maximized too.

Reply:
Yes, it does.